src/hb-aat-layout.cc

```diff
@@ -341,10 +376,12 @@ hb_aat_layout_get_feature_types (hb_face_t                    *face,
 
 /**
  * hb_aat_layout_feature_type_get_name_id:
- * @face: a face object
- * @feature_type: feature id
+ * @face: #hb_face_t to work upon
+ * @feature_type: The #hb_aat_layout_feature_type_t of the requested feature type
  *
- * Return value: Name ID index
+ * Fetches the name ID of the specified feature type in the face's 'name' table.
```

Comment:
```suggestion
 * Fetches the name ID of the specified feature type in the face's `name` table.
```
For consistency with other occurrences of table names.